tests/test_dle.py

```diff
@@ -318,6 +318,60 @@ def test_structure_2p5D(self):
             assert iters[0].step == par_tile[1]
             assert iters[1].step == par_tile[0]
 
+    
+    @pytest.mark.parametrize('par_tile,expected', [
+        ((32, 4, 4), ((4, 4, 32), (4, 4, 32))),
+        (((16, 4, 4), (16, 16, 16)), ((4, 4, 16), (16, 16, 16))),
+        (((32, 4, 4), 1), ((4, 4, 32), (4, 4, 32))),
+        (((32, 4, 4), 1, 'tag0'), ((4, 4, 32), (4, 4, 32))),
+        ((((32, 4, 8), 1, 'tag0'), ((32, 8, 4), 2)), ((8, 4, 32), (4, 8, 32))),
+    ])
+    def test_openacc_structure(self, par_tile, expected):
```

Comment:
Sorry, I made a mistake; clearly these tests belong to `test_gpu_openacc`, not to `test_dle` which constitutes a platform-independent set of tests

Reply:
(By moving them there you'll have to rename them as well -- dropping "openacc" for example)